astropy/io/votable/tests/test_table.py

```diff
@@ -52,6 +52,49 @@ def home_is_tmpdir(monkeypatch, tmp_path):
     monkeypatch.setenv("USERPROFILE", str(tmp_path))
 
 
+def test_read_write_votable_parquet(tmp_path):
```

Comment:
I suggest to move this test further down in the file. I would also add a read only test after putting small files in the `data` subdirectory.

Reply:
moved it down a bit.
Reading only is currently an issue because the path linking the VOTable with the Parquet file is hard coded (upon saving the file by user).

Reply:
Added a small test file, which in turn also tests that the relative path works.